doc/community-build/docker/cortx-all/README.md

```diff
@@ -6,17 +6,17 @@ This document provides step-by-step instructions to build required binaries and
 
 - Do not update OS or kernel package with yum update as the kernel version must be set to 3.10.0-1160.el7
 - Do not upgrade packages from CentOS 7.8 to CentOS 7.9
-- Docker >= 20.10.10 . Please refer [Install Docker Community Edition](https://docs.docker.com/engine/install/centos/) steps. Validate docker version on system. 
+- Before installing docker, docker home directory space should be 70GB (default docker home directory is /var/lib/docker) and /mnt drive space should be 30GB to run cortx build.
```

Comment:
We need Docker 20.10 to use the latest capabilities. 
```
[root@ssc-vm-rhev4-0707 ~]# docker --version
Docker version 20.10.8, build 3967b7d
[root@ssc-vm-rhev4-0707 ~]#
[root@ssc-vm-rhev4-0707 ~]# rpm -qa docker-ce
docker-ce-20.10.8-3.el7.x86_64
[root@ssc-vm-rhev4-0707 ~]#
```

Reply:
done